src/api/Gemfile.next.lock

```diff
@@ -0,0 +1,559 @@
+GEM
```

Comment:
Hmmm, how are we keeping this in sync and up to date?

Reply:
I have to figure this out. It should be possible to run Depfu on Gemfile.next. This has to be set after the PR is merged.